### DISCUSSION TITLE
Command line parameter to specify html comparison report for each of the files in the folder

### DISCUSSION THREAD

**Discussion by Meena1983** [WinMerge/winmerge#2034] [Q&A] [answered]
Dear all,

I am trying to write a bat file to compare the files between 2 folders and provide a html comparison report. But if I specify -or /or parameter, it writes only one comparison report for all the files compared and says "same" or "different". How do I see the report for each of the files getting compared in the colorful html view like we can see in the tools -> generate html report? Are there any options available?

**Comment by sdottaka** [comment, marked as answer]
If you check `Include File Compare Report` and press OK as shown below, file difference reports will also be output. However, only files with differences are created.
This selection also affects when you create a report from the command line.

![image](https://github.com/WinMerge/winmerge/assets/98126/26f8fada-cd5b-4d40-b815-23c3b984e316)

Although not documented, you can also have the same effect as above by specifying the following as a command line argument:

` -cfg ReportFiles/IncludeFileCmpReport=1`

**Reply by Meena1983** [reply]
Thank you for your quick response. 
I checked with this option in the bat file. But if it is in noninteractive mode, it does not work. I had to click on generate report again in the UI where it presented the comparison view. I think it is better to go for single file comparison along with -or option for my usecase. That way I would be able to use unpacker and also get the report separately. I will find a way to get all files in the folders and compare it one by one. Thank you again!
